### PR TITLE
Add support for format css

### PR DESCRIPTION
**Pull request by isbasex**

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -2,21 +2,26 @@ const prettier = require('prettier')
 
 function matchedLang(lang) {
   lang = lang.trim()
-  return ['js', 'javascript', 'jsx'].indexOf(lang) > -1
+  return ['js', 'javascript', 'jsx', 'css', 'scss'].indexOf(lang) > -1
```
Comment (egoist) [inline index.js]: seems less is also supported

File: index.js [hunk 2]
```diff
@@ -2,21 +2,26 @@ const prettier = require('prettier')
...
 }
 
-module.exports = function (md, opts) {
+function getParser (lang) {
+  return ['css', 'scss'].indexOf(lang.trim()) > -1 ? 'postcss' : 'babylon'
+}
+
+module.exports = function (md, opts, _prettier = prettier) {
```
Comment (egoist) [inline index.js]: we can move `prettier` to `peerDependencies` then we won't need this

**Comment by isbasex** [comment]
@egoist All right

**Comment by egoist** [comment]
`v1.1.0` now supports all parsers in Prettier.

**Pull request closed without merging**